### ISSUE TITLE
update-badge adds new badge even when one is present

### ISSUE THREAD

**Issue by markgoho** [bahmutov/check-code-coverage#19]
Current README:
``` md
![check-code-coverage](https://img.shields.io/badge/code--coverage-92.9%25-brightgreen)
![Master](https://github.com/dchbx/enroll-ui/workflows/Test,%20Lint,%20Build/badge.svg?branch=master)
```

Run `DEBUG=check-code-coverage npx update-badge --from /Users/markgoho/Projects/enroll-ui/apps/admin-e2e/coverage/coverage-summary.json`

``` bash
check-code-coverage args: { _: [], '--from': '/Users/markgoho/Projects/enroll-ui/apps/admin-e2e/coverage/coverage-summary.json' } +0ms
  check-code-coverage reading coverage from { filename: '/Users/markgoho/Projects/enroll-ui/apps/admin-e2e/coverage/coverage-summary.json' } +0ms
  check-code-coverage reading coverage summary from: /Users/markgoho/Projects/enroll-ui/apps/admin-e2e/coverage/coverage-summary.json +0ms
  check-code-coverage clamped coverage: 92.9 +3ms
  check-code-coverage coverage regex: "/https:\/\/img\.shields\.io\/badge\/code--coverage-\d+%25-(:?red|yellow|green|brightgreen)/" +1ms
  check-code-coverage for coverage 92.9% badge color "brightgreen" +1ms
  check-code-coverage new coverage badge: "https://img.shields.io/badge/code--coverage-92.9%25-brightgreen" +0ms
⚠️ Could not find code coverage badge in file /Users/markgoho/Projects/enroll-ui/README.md
Insert new badge on the first line
  check-code-coverage inserting new badge: ![check-code-coverage](https://img.shields.io/badge/code--coverage-92.9%25-brightgreen) +0ms
saving updated readme with coverage 92.9%
```

Updated README:
``` md
![check-code-coverage](https://img.shields.io/badge/code--coverage-92.9%25-brightgreen) ![check-code-coverage](https://img.shields.io/badge/code--coverage-92.9%25-brightgreen)
![Master](https://github.com/dchbx/enroll-ui/workflows/Test,%20Lint,%20Build/badge.svg?branch=master)
```

P.S. ❤️ the debugging, it helps so much!

**Comment by bahmutov** [comment]
I bet it does not handle fractional part in percentage, could you check the regex there

**Comment by v1b3m** [comment]
Hey,
Just asking whether this issue has been resolved in the latest npm package cause I'm still getting it.
```
![check-code-coverage](https://img.shields.io/badge/code--coverage-66.51%25-yellow) ![check-code-coverage](https://img.shields.io/badge/code--coverage-66.51%25-yellow) ![check-code-coverage](https://img.shields.io/badge/code--coverage-66.51%25-yellow) ![check-code-coverage](https://img.shields.io/badge/code--coverage-66.51%25-yellow)
```
Thanks.

**Comment by bahmutov** [comment]
@v1b3m could you open a new issue for this - we seem to not handle double percentage after the dot.

**Comment by v1b3m** [comment]
> tada This issue has been resolved in version 1.9.3 tada
> 
> The release is available on:
> 
>     * [npm package (@latest dist-tag)](https://www.npmjs.com/package/check-code-coverage/v/1.9.3)
> 
>     * [GitHub release](https://github.com/bahmutov/check-code-coverage/releases/tag/v1.9.3)
> 
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot packagerocket

Hey @bahmutov,
Thanks for the release, I've updated the node package and it is now working perfectly. Thanks again.
